.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -3,30 +3,21 @@
 
 closes #
 
-### Short Description and Why It's Useful
+### Short Description of the PR
 <!-- describe in a few words what is this Pull Request changing and why it's useful -->
 
 
-
 ### Screenshots of Visual Changes before/after (if There Are Any)
 <!-- if you made any changes in the UI layer please provide before/after screenshots -->
 
-### Which Environment This Relates To
-Check your case. In case of any doubts please read about [Release Cycle](https://docs.vuestorefront.io/guide/basics/release-cycle.html)
-
-- [ ] Test version (https://test.storefrontcloud.io) - this is a new feature or improvement for Vue Storefront. I've created branch from `develop` branch and want to merge it back to `develop`
-- [ ] RC version (https://next.storefrontcloud.io) - this is a stabilisation fix for Release Candidate of Vue Storefront. I've created branch from `release` branch and want to merge it back to `release`
-- [ ] Stable version (https://demo.storefrontcloud.io) - this is an important fix for current stable version. I've created branch from `hotfix` or `master` branch and want to merge it back to `hotfix`
-
-### Upgrade Notes and Changelog
-- [x] No upgrade steps required (100% backward compatibility and no breaking changes)
-- [ ] I've updated the [Upgrade notes](https://github.com/DivanteLtd/vue-storefront/blob/develop/docs/guide/upgrade-notes/README.md) and [Changelog](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md) on how to port existing Vue Storefront sites with this new feature
-- [ ] (Next only) I've followed this [instruction](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and I've created a `.js` file with information about my Pull Request
-
-**IMPORTANT NOTICE** - Remember to update `CHANGELOG.md` with description of your change
+### Pull Request Checklist
+<!-- we will not merge your Pull Request until all checkboxes are checked -->
+- [ ] I have updated the Changelog ([V1](https://github.com/DivanteLtd/vue-storefront/blob/develop/CHANGELOG.md)) [v2](https://docs-next.vuestorefront.io/contributing/creating-changelog.html) and mentioned all breaking changes in the public API.
+- [ ] I have documented all new public APIs and made changes to existing docs mentioning the parts I've changed so they're up to date.
```

Comment:
In line below you are using full forms. Let's unify it and either use full forms in whole file or not.